Makefile

```
@@ -64,8 +65,29 @@ proxy-h:
 proxy-m:
 	consul connect proxy -service minio-local -upstream minio:9000 -log-level debug
 # to-postgres
-proxy-p:
+proxy-pg:
 	consul connect proxy -service postgres-local -upstream postgres:5432 -log-level debug
+
 # to-presto
-proxy-p:
-	consul connect proxy -service postgres-local -upstream presto:8080 -log-level debug
+proxy-presto:
+	consul connect proxy -service presto-local -upstream presto:8080 -log-level debug
+
+download-presto-cli:
+	curl https://repo1.maven.org/maven2/io/prestosql/presto-cli/${PRESTO_VERSION}/presto-cli-${PRESTO_VERSION}-executable.jar -o ./${presto}
+	chmod +x ./${presto}
+
+presto-cli:
+ifeq (,$(wildcard ./${presto}))
+	$(MAKE) download-presto-cli
+endif
+	${presto} --server localhost:8080 --catalog hive --schema default --user presto
+
+# manual table creation for different file types
+csv:
```

Shouldn't *csv*, *json*, *avro* targets be dependent on *proxy-presto* and *presto-cli* targets?
```suggestion
csv: proxy-presto presto-cli
```

---

Tables creation is automated via ansible. These commands are manual examples. And again, proxy's stdout is attached to the terminal. I would suggest running `presto-cli` in separate terminal.

---

I guess we should move them out from **Makefile** and add it in the documentation -> example commands for manual table creation for different file types